mod_tokens.asciidoc

```diff
@@ -483,4 +482,4 @@ the state of the Token might have changed.
 |ALLOWED |It is allowed to whitelist the token, <<mod_tokens_real-time_authorization,realtime authorization>> is also allowed. The CPO may choose which version of authorization to use.
 |ALLOWED_OFFLINE |In normal situations <<mod_tokens_real-time_authorization,realtime authorization>> shall be used. But when the CPO cannot get a response from the eMSP (communication between CPO and eMSP is offline), the CPO shall allow this Token to be used.
 |NEVER |Whitelisting is forbidden, only <<mod_tokens_real-time_authorization,realtime authorization>> is allowed. CPO shall always send a <<mod_tokens_real-time_authorization,realtime authorization>> for any use of this Token to the eMSP.
-|===
\ No newline at end of file
+|===
```

Comment:
What did you really change here? Watch out with making changes to the layout of tables, sometimes it breaks the layout.

Reply:
I changed a couple of lines:

NOTE: If real-time authorization is asked for a location, the eMSP SHALL NOT validate that charging is possible based on information like opening hours or EVSE status etc. as this information might not be up to date.

and

|country_code |<<types.asciidoc#types_cistring_type,CiString>>(2) |yes |Country code of the eMSP sending this PUT request to the CPO system. This SHALL be the same value as the `country_code` in the Token object being pushed.

The second one is in a table, but all I did was take out a rogue ` character on the word "Token".  I did not purposefully modify the end of the file.  You'd think that if git was so fussy about that, they'd be really careful NOT to purposefully mess up files as people were checking them in.

Is this something you can fix or do I need to make another pull-request?